### ISSUE TITLE
[FEATURE_REQUEST] Add an option to keep all character cards in context during group chats

### ISSUE THREAD

**Issue by ClassicDirt** [SillyTavern/SillyTavern#1268]
### Have you searched for similar requests?

Yes

### Is your feature request related to a problem? If so, please describe.

When using the group chat feature currently, whenever a participant sends a message, their card is loaded at the beginning of the context. This makes the entire context have to be reprocessed with each message. When using koboldcpp, this makes group chats unbearably slow.

### Describe the solution you'd like

Keep all participant's cards loaded into the context, modifying only the end of the prompt to denote which character the model should send a message as. While this extends context length, it significantly speeds up generation times. Given the trade off, adding a toggle option may be the best approach.

### Priority

Medium (Would be very useful)

### Is this something you would be keen to implement?

No

**Comment by Cohee1207** [comment]
Duplicate of #1231

TL;DR: I advise using group chats with members' info added as lorebook entries. Implementing this feature as requested basically requires a total rewrite of the prompt building to allow multiple descriptions per prompt or a complete trashing of the current group chat implementation.

**Comment by valden80** [comment]
@Cohee1207 
May I add IMHO related to this solution? 
I already using it this way, but in this case ability to bind multiple lorebooks (not just single) to group chat becoming very important from the ergonomic point. With only single lorebook you need manually squash all data of all characters for chat in it, and can't easily add "modules" - data about RP world related to only this chat.

**Comment by Cohee1207** [comment]
Added to release 1.10.6
https://docs.sillytavern.app/usage/core-concepts/groupchats/#join-character-cards
